contrib/ansible/Getting-Started.md

```diff
@@ -0,0 +1,157 @@
+
+
+# Kubernetes Cluster with Containerd and Cri-Containerd
+
+In this document we aim to provide simple steps to bring up a Kubernetes cluster using ansible and kubeadm tools. 
+
+### Prerequisites:
+- **OS**: Ubuntu 16.04 (Other OS installations will be validated and updated)
+- **Python**: 2.7+
+- **Ansible**: 2.3+
+
+## Step 0:
+-  Install Ansible on any one of the nodes in the candidate cluster or an external machine which will be used to provision the cluster. The install instructions can be found [here](http://docs.ansible.com/ansible/latest/intro_installation.html). The playbooks in the repo need ansible version 2.3 or later.
```

Comment:
Install Ansible on the host where you will provision the cluster. This host may be one of the nodes you plan to include in your cluster. Installation instructions for Ansible are found

Reply:
/The playbooks in the repo need ansible version 2.3 or later.//
Not needed since you already said 2.3 above..

Reply:
Is provision correct here? We don't actually create the VM.

May be problem with my English.

Reply:
I would see commissioning a node as creating VM and provisioning as installing on a VM. We can change it if its confusing.

Reply:
It's your call. You know my English. :)
I always hear provisioning a node when creating a new VM in GCE, thus ask here.

Reply:
provisioning is correct.. it means installing and configuring required software